### PR TITLE
Added jsexpr function

### PR DESCRIPTION
**Pull request by leandrogehlen**
| Q             | A
| ------------- | ---
| Is bugfix?    | no
| New feature?  | yes
| Breaks BC?    | no
| Tests pass?   | yes

### CODE REVIEW

File: src/Extension.php [hunk 1]
```diff
@@ -45,6 +46,7 @@ public function __construct($viewRenderer, $smarty)
         $smarty->registerPlugin('function', 'url', [$this, 'functionUrl']);
         $smarty->registerPlugin('function', 'set', [$this, 'functionSet']);
         $smarty->registerPlugin('function', 'meta', [$this, 'functionMeta']);
+        $smarty->registerPlugin('function', 'jsexpr', [$this, 'functionJsExpr']);
```
Comment (samdark) [inline src/Extension.php]: How about naming it just `js`?

Comment (leandrogehlen) [reply src/Extension.php]: i really dont know. You can decide it.

Suggestions: 
- `js`
- `javascript`
- `jsexpr`

Comment (samdark) [reply src/Extension.php]: `js` is short and still pretty much understandable.

File: src/Extension.php [hunk 2]
```diff
@@ -252,6 +254,31 @@ public function functionMeta($params, $template)
         Yii::$app->getView()->registerMetaTag($params, $key);
     }
 
+     /**
```
Comment (samdark) [inline src/Extension.php]: Formatting is off.

**Comment by samdark** [comment]
How about adding tests for it and a entry for changelog?

**Comment by samdark** [comment]
Merged. Thank you!

**Pull request merged**